core/chains/starknet/orm_test.go

```diff
@@ -0,0 +1,116 @@
+// TODO: Improve code reuse (mostly c/p of core/chains/terra/orm_test.go)
+package starknet_test
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/sqlx"
+
+	"github.com/smartcontractkit/chainlink-starknet/relayer/pkg/chainlink/db"
+
+	"github.com/smartcontractkit/chainlink/core/chains/starknet"
+	"github.com/smartcontractkit/chainlink/core/chains/starknet/types"
+	"github.com/smartcontractkit/chainlink/core/internal/testutils/pgtest"
+	"github.com/smartcontractkit/chainlink/core/logger"
+)
+
+func setupORM(t *testing.T) (*sqlx.DB, types.ORM) {
+	t.Helper()
+
+	db := pgtest.NewSqlxDB(t)
+	orm := starknet.NewORM(db, logger.TestLogger(t), pgtest.NewPGCfg(true))
+
+	return db, orm
+}
+
+func Test_ORM(t *testing.T) {
+	_, orm := setupORM(t)
+
+	dbcs, err := orm.EnabledChains()
+	require.NoError(t, err)
+	require.Empty(t, dbcs)
+
+	chainIDA := fmt.Sprintf("Chainlinktest-%d", rand.Int31n(999999))
+	_, err = orm.CreateChain(chainIDA, &db.ChainCfg{})
+	require.NoError(t, err)
+	chainIDB := fmt.Sprintf("Chainlinktest-%d", rand.Int31n(999999))
+	_, err = orm.CreateChain(chainIDB, &db.ChainCfg{})
+	require.NoError(t, err)
+
+	dbcs, err = orm.EnabledChains()
+	require.NoError(t, err)
+	require.Len(t, dbcs, 2)
+
+	newNode := db.Node{
+		Name:    "first",
+		ChainID: chainIDA,
+		URL:     "http://tender.mint.test/columbus-5",
+	}
+	gotNode, err := orm.CreateNode(newNode)
+	require.NoError(t, err)
+	assertEqual(t, newNode, gotNode)
+
+	gotNode, err = orm.Node(gotNode.ID)
+	require.NoError(t, err)
+	assertEqual(t, newNode, gotNode)
+
+	newNode2 := db.Node{
+		Name:    "second",
+		ChainID: chainIDB,
+		URL:     "http://tender.mint.test/bombay-12",
```

Comment:
Bombay-12? :) To avoid confusion, I'd suggest we either use dummy data here or StarkNet specific.